docs/GettingStartedDocs/Contributors/BuildingInADockerContainer.md

```diff
@@ -9,11 +9,11 @@
 1. Install Docker CE by following these instructions: https://docs.docker.com/install/linux/docker-ce/ubuntu/#install-docker-ce
 
 2. Install the appropriate Intel SGX Driver for your platform. For example, if you're running on an SGX1 with FLC system, you'll probably want to install the Intel SGX DCAP Driver for your platfrom: https://01.org/intel-software-guard-extensions/downloads
-    - If you're running on an SGX with FLC system, you'll probably want to install the Intel SGX DCAP Driver for your platfrom: https://01.org/intel-softwareguard-extensions/downloads/intel-sgx-dcap-linux-1.1-release
+    - If you're running on an SGX with FLC system, you'll probably want to install the Intel SGX DCAP Driver for your platfrom: https://01.org/intel-softwareguard-extensions/downloads/intel-sgx-dcap-linux-1.2-release
     - If you're running on an SGX without FLC system, you'll want to install the Intel SGX Driver for your platform: https://01.org/intel-softwareguard-extensions/downloads/intel-sgx-linux-2.5-release
```

Comment:
This should be bumped up to the 2.6 release ... we should also test that this works correctly in the manual scenario since our AESM prereqs script still pulls from the latest and we don't automate testing around that now.

Reply:
Thanks Simon, good advice.  I'll test that out right now on my local SGX enabled VM

Reply:
I've updated it accordingly! In fact, I had to add some more information to describe how to get the AESM scenario working in a container :)

Also, thanks for reminding me about that prereqs directory. We need to figure out what to do with that eventually. I'm going to file an issue for it